src/main/java/net/neoforged/neoforge/common/conditions/ConditionContext.java

```diff
@@ -17,6 +17,7 @@
 
 public class ConditionContext implements ICondition.IContext {
     private final TagManager tagManager;
+    // TODO: Clear loaded tags after reloads complete. The context object may leak, but we still want to invalidate it.
```

Comment:
if you don't add `1.20.5` somewhere in the TODO we'll never find it when porting. :P

Reply:
Forgot to add `1.20.5`?

Reply:
Oh, I added it to a different todo comment but not this one